src/nvim/mouse.c

```diff
@@ -1133,6 +1122,24 @@ bool is_mouse_key(int c)
          || c == K_X2RELEASE;
 }
 
+static void mouse_tab_close(int c1)
```

Comment:
I think this function should be moved below `move_tab_to_mouse()` so that it is grouped with more similar functions.